modules/templates-writing-description.adoc

```diff
@@ -58,13 +58,7 @@ constants file.
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 The categories can also be customized for the whole cluster.
 endif::[]
-<6> An icon to be displayed with your template in the web console. Choose from
-our existing logo icons when possible. You can also use icons from FontAwesome.
-ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
-Alternatively, provide icons through CSS customizations that can be added to an
-{product-title} cluster that uses your template. You must specify an icon class
-that exists, or it will prevent falling back to the generic icon.
-endif::[]
+<6> An icon to be displayed with your template in the web console. 
```

Comment:
We need to give users some guidance on what icons are available. Are we able to link to this Git repo?

https://github.com/openshift/openshift-logos-icon

Reply:
@sferich888 Are you okay with adding that link?

Reply:
@rhamilto Is this Git repo still the right place to look for icons?

Reply:
> @rhamilto Is this Git repo still the right place to look for icons?

It's complicated.  It is but it isn't because we switched to [image replacement of the font-based icons](https://github.com/openshift/console/blob/master/frontend/public/components/catalog/catalog-item-icon.tsx#L99-L195) from https://github.com/openshift/openshift-logos-icon some time ago.  But the legacy API means users still need to use the class names from the font-based icon set (e.g., `icon-infinispan`).  So maybe we say to look at https://github.com/openshift/console/tree/master/frontend/public/imgs/logos and prefix any of those image names with `icon-` and omit the file extension instead?

Reply:
Maybe we just list them here in the docs? The icons don't change often.

Reply:
@spadgett I think that might be the best path forward.